pom.xml

```diff
@@ -136,8 +136,9 @@
         <jackson.version>2.10.3</jackson.version>
         <jackson.databind.version>2.10.0</jackson.databind.version>
         <kafka.version>2.4.1</kafka.version>
-        <janus.version>0.4.1</janus.version>
+        <janus.version>0.5.0</janus.version>
         <gremlin.version>3.4.6</gremlin.version>
+        <netty.version>4.1.42.Final</netty.version>
```

Comment:
I note netty latest is 45.1.48-Final . I expect dependabot will point this out later. Is there a reason for not going there?

Hopefully we'll fix #2669 but I will pull & check

Reply:
I selected 4.1.42.Final only because it seems to be what JG, Cassandra et al have been tested with. We could push to the latest netty update (4.1.48.Final) if you prefer.

Reply:
Either... ok as is, but I will plan to look at #2669 straight away and pull up there?